packages/react/src/components/AccountSettings/DeleteUser/__tests__/DeleteUser.test.tsx

```diff
@@ -15,6 +23,29 @@ jest.mock('../../../../internal', () => ({
 
 const deleteUserSpy = jest.spyOn(UIModule, 'deleteUser');
 
+function CustomWarning({ onCancel, onConfirm, isDisabled }) {
+  return (
+    <Flex direction="column">
+      <Text variation="warning">Custom Warning Message</Text>
+      <Button onClick={onCancel}>Back</Button>
+      <Button variation="primary" onClick={onConfirm} isDisabled={isDisabled}>
+        Custom Confirm Button
+      </Button>
+    </Flex>
+  );
+}
+
+const components: DeleteUserComponents = {
+  SubmitButton: (props) => <Button {...props}>Custom Delete Button</Button>,
+  Warning: (props) => <CustomWarning {...props} />,
```

Comment:
Should be able to just write this as:
```suggestion
  Warning: CustomWarning,
```

Reply:
Done in [bb9561a](https://github.com/aws-amplify/amplify-ui/pull/3123/commits/bb9561af5a297d4dc460d1d456dbac54bfeae3cd) 😈